roiextractors/extractors/schnitzerextractor/cnmfesegmentationextractor.py

```diff
@@ -55,23 +52,23 @@ def _tot_exptime_extractor_read(self):
         return self._dataset_file[self._group0[0]]['time']['totalTime'][0][0]
 
     def _summary_image_read(self):
-        summary_images_ = self._dataset_file[self._group0[0]]['Cn']
-        return np.array(summary_images_).T
+        summary_image = self._dataset_file[self._group0[0]]['Cn']
+        return np.array(summary_image).T
```

Comment:
shouldn't this be the correlation image?

Reply:
i'm calling this summary image in all classes, but its assigned to `self._image_correlation` in the `__init__`